src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -218,6 +219,26 @@ def put(
                 status=403,
             )
 
+        is_member = not (
+            request.access.has_scope("member:invite") and request.access.has_scope("member:admin")
+        )
+        enable_member_invite = (
+            features.has("organizations:members-invite-teammates", organization)
+            and not organization.flags.disable_member_invite
+        )
+        # Members can only resend invites
+        reinvite_request_only = set(result.keys()).issubset({"reinvite", "regenerate"})
+        # Members can only resend invites that they sent
+        is_invite_from_user = member.inviter_id == request.user.id
+
+        if is_member:
+            if not enable_member_invite:
+                return Response({"detail": ERR_INSUFFICIENT_SCOPE}, status=403)
```

Comment:
The err msg `ERR_INSUFFICIENT_SCOPE` mentions requiring `member:admin` which is not true, so we can just raise a generic err
```suggestion
                raise PermissionDenied
```